rpc_jobs/pull_request_issue_link.yml

```diff
@@ -0,0 +1,31 @@
+- job:
```

Comment:
I think this needs to be a job-template so we can apply this to multiple repos. To start with rpco and rpc-gating would be useful.